terraform/development/shared/main.tf

```diff
@@ -0,0 +1,168 @@
+terraform {
+  required_version = "~>1.5.1"
+
+  required_providers {
+    aws = {
+      version = "~>5.0"
+      source  = "hashicorp/aws"
+    }
+  }
+
+  backend "s3" {
+    bucket         = "core-non-prod-tf-state"
+    dynamodb_table = "core-non-prod-tf-state-lock"
+    encrypt        = true
+    key            = "core-development.tfstate"

```

Comment:
Do we perhaps want to call this core-shared instead?

Reply:
Would have been nice to but as there's some state tracking already in `core-development.tfstate`, we'd have to migrate the state to `core-shared.tfstate` I think, which is pretty annoying. Especially as the state being tracked currently has what we created to enable us to inform the Dev DNS changes required, so probably not worth it imo 😀

Reply:
Doesn't sound too hard (see [here](https://stackoverflow.com/questions/63464775/terraform-renaming-state-file)) and if done correctly shouldn't affect any of the actual infra AFAICT. But happy to leave as is (or perhaps change later) given it's not critical path.

Reply:
I've managed to make these changes and migrate the state 🎉, thanks!

We now have `core-development-shared.tfstate` and `core-development-per-review-app.tfstate`